fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -29,20 +30,23 @@ const RollCallOpened = () => {
   const route = useRoute();
   const { rollCallID, time } = route.params;
   const navigation = useNavigation();
-  const [, setQrWasScanned] = useState(false);
-  const [attendeesSet, updateAttendeesSet] = useState(new Set<string>());
```

Comment:
Careful, a `Set` will automatically avoid de-duplication, whereas a list will allow you to scan 3 times the same key.
You might want to add a test to verify/document this behaviour, actually 😄